### DISCUSSION TITLE
How to add custom PopUpContextMenu on Custom popup bar

### DISCUSSION THREAD

**Discussion by pareshios** [LeoNatan/LNPopupController#576] [General] [answered]
I need to show an error view on top of PopUpBar. Is there any way to add it using context menu?.

**Comment by LeoNatan** [comment]
Please do not open issues for questions. I also do not understand what you are asking.

**Reply by LeoNatan** [reply, marked as answer]
You can add `UIContextMenuInteraction` directly to `LNPopupBar`:

![image](https://github.com/user-attachments/assets/7c96d0b2-6b74-4bdc-98e6-30aaff2c8e3c)

If you have a custom bar, you can still add it to the popup bar, or to a specific view inside your view hierarchy. SwiftUI should work too.
